repository.json

```diff
@@ -512,7 +512,7 @@
 			],
 			"releases": [
 				{
-					"sublime_text": ">=4095",
+					"sublime_text": ">=4070",
```

Comment:
This is not possible to change to 4070. I put this on purpose to 4095 because LSP-julia uses [`sublime.ListInputItem`](https://www.sublimetext.com/docs/api_reference.html#sublime.ListInputItem) which was introduced with ST 4095.

The 4070 restriction for the main LSP package is mostly meaningless anyway, because I'm almost sure it uses various things from the ST API that were introduced after build 4070. Personally I also don't like the `4070-` prefix for the version tags (purely for aesthetic reasons). iirc it was introduced to make it possible to add new release prefixes with increased build numbers whenever new functionality from the ST API gets used in LSP. But apparently this didn't happen in the past and also nobody really cares about it, so I guess it can just stay like this (4070) forever and in the case of any issue reports get filed in the future for the use of API incompatible with older ST builds, we could just fix those gradually with `if ST_VERSION > XXXX` checks in the code, if possible.

Reply:
Thanks for pointing that out.

I went through the LSP-* packages again
and checked if other packages use `sublime.*` apis introduced in later versions of ST and reverted the versions.